ci/scripts/trigger_publish_artifacts.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# Copyright the Hyperledger Fabric contributors. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+docker login -u "${ARTIFACTORY_USERNAME}" -p "${ARTIFACTORY_PASSWORD}" hyperledger-fabric.jfrog.io
+
+for image in baseos peer orderer ccenv tools ca javaenv nodeenv; do
+    docker tag "hyperledger/fabric-${image}" "hyperledger-fabric.jfrog.io/fabric-${image}:amd64-${RELEASE}"
+    docker push "hyperledger-fabric.jfrog.io/fabric-${image}:amd64-${RELEASE}"
+
+    docker tag "hyperledger/fabric-${image}" "hyperledger-fabric.jfrog.io/fabric-${image}:amd64-latest"
+    docker push "hyperledger-fabric.jfrog.io/fabric-${image}:amd64-latest"
+done
+
+cd "${GOPATH}/src/github.com/hyperledger/fabric/release/linux-amd64/"
+mkdir "config"
```

Comment:
I prefer to use option to create it if necessary, but proceed ok if it exsits (-p), to avoid potential early exit when using mode "-e" (unless you really want to exit if config dir already exists).
mkdir -p "config"

Reply:
agreed, it was an oversight, great catch

Reply:
good catch, agreed